javascript-modal/index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Javascript Modal</title>
+    <link rel="stylesheet" href="styles.css">
+  </head>
+  <body>
+    <div class="container ">
+      <div class="row">
+        <div class="col-full">
+          <div class="absolute hidden">
+            <div class="overlay"></div>
+            <div class="survey-modal">
+              <div class="white-box">
+                <h2 class="text-style">Would you like to take a survey?</h2>
+                <button class="button-no">NO</button>
+              </div>
+            </div>
+          </div>
```

Comment:
I would move the HTML for the modal to the bottom of the `body`, right before the `script` element. It doesn't end up staying in the document flow because it is `absolute` positioned, so keeping it separate from the content makes sense.